action.yml

```diff
@@ -34,6 +34,7 @@ runs:
       with:
         payload: |
           {
+            "icon_url": "https://files.slack.com/files-tmb/T02P98BKE-F056Q4DK0D7-3a0e728b4d/image_from_ios_360.jpg",
```

Comment:
I don't think this URL is accessible if you're not logged in. Can we upload the icon to this GitHub repository, and use a `raw.githubusercontent.com` link?

Reply:
we're only ever going to see the image on Slack... so that should be fine? non?

that said I kinda loathe the uri so I went ahead and added it. much cleaner

Reply:
It will likely only be used in our Slack, yes. But it would also be deleted automatically at some point I think? And in case anyone else is using this action outside of MetaMask and using the Slack feature, it's better to include it in the repo.

Reply:
that is a very good point.

Reply:
Given that other people are using this action for their own projects, it would be great if we could dynamically set the icon used. That way other projects aren't stuck with our metamaskbot icon.